tasks/components/layout/TaskTemplateDisplay.tsx

```diff
@@ -0,0 +1,93 @@
+import { tw } from '@helpwave/common/twind'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { LucideArrowLeftRight } from 'lucide-react'
+import { TaskTemplateCard } from '../cards/TaskTemplateCard'
+import { Button } from '@helpwave/common/components/Button'
+import { AddCard } from '../cards/AddCard'
+import { useRouter } from 'next/router'
+import type { TaskTemplateDTO } from '../../mutations/task_template_mutations'
+import { Span } from '@helpwave/common/components/Span'
+
+export type TaskTemplateDisplayTranslation = {
+  addNewTaskTemplate: string,
+  switchToPersonal: string,
+  switchToWardView: string,
+  taskTemplates: string
+}
+
+const defaultTaskTemplateDisplayTranslation = {
+  en: {
+    addNewTaskTemplate: 'Add new template',
+    switchToPersonal: 'Personal Task Templates',
+    switchToWardView: 'Ward Task Templates'
+  },
+  de: {
+    addNewTaskTemplate: 'Neue Vorlage hinzufügen',
+    switchToPersonal: 'Persönliche Task Vorlagen',
+    switchToWardView: 'Stations Task Vorlagen'
+  }
+}
+
+export type TaskTemplateDisplayProps = {
+  selectedID: string,
+  onSelectChange: (taskTemplate: TaskTemplateDTO | undefined) => void,
+  taskTemplates: TaskTemplateDTO[],
+  variant: 'wardTemplates' | 'personalTemplates'
+}
+
+/**
+ * A column for showing TaskTemplates either for Ward or Private templates
+ */
+export const TaskTemplateDisplay = ({
+  language,
+  selectedID,
+  onSelectChange,
+  taskTemplates,
+  variant
+}: PropsWithLanguage<TaskTemplateDisplayTranslation, TaskTemplateDisplayProps>) => {
+  const translation = useTranslation(language, defaultTaskTemplateDisplayTranslation)
+  const router = useRouter()
+  const { uuid } = router.query
+
+  return (
+    <div className={tw('py-4 px-6')}>
+      <div className={tw('flex flex-row items-center justify-between mb-4')}>
+        <Span type="subsectionTitle">
+          {variant === 'personalTemplates' ? translation.switchToPersonal : translation.switchToWardView}
```

Comment:
This is an anti-pattern. Rename the translation!

Reply:
Changing the name makes sense, because it's not accurate. What exactly is the antipattern here?

Reply:
Abusing translations is an antipattern.

Reply:
There are two separate translations; each should get its key (we had this conversation with @JannikWibker before)

So is the ternary operator the problem, or what exactly is it?

Reply:
`translation.switchToPersonal` is used as `translation.personalTitle` or something like that

Reply:
yes, so it's a problem of the translation key implying another use, so renaming it to `personalTaskTemplates` would solve this?